packages/synthetix-main/contracts/modules/SNXTokenModule.sol

```diff
@@ -32,7 +31,7 @@ contract SNXTokenModule is ISNXTokenModule, OwnableMixin, SNXTokenStorage {
 
         store.snxTokenAddress = snxTokenProxyAddress;
 
-        emit SNXTokenCreated(snxTokenProxyAddress);
+        emit SatelliteCreated("contracts/token/SNXToken.sol:SNXToken", address(snxTokenProxyAddress));
```

Comment:
We are going to use just the contract name instead of the fullyQualifiedName: https://github.com/Synthetixio/synthetix-v3/issues/718